base/strings/substring.jl

```diff
@@ -107,7 +107,7 @@ pointer(x::SubString{String}, i::Integer) = pointer(x.string) + x.offset + (i-1)
 Reverses a string. Technically, this function reverses the codepoints in a string and its
 main utility is for reversed-order string processing, especially for reversed
 regular-expression searches. See also [`reverseind`](@ref) to convert indices in `s` to
-indices in `reverse(s)` and vice-versa, and [`Unicode.graphemes`](@ref Base.Unicode.graphemes) to
+indices in `reverse(s)` and vice-versa, and `graphemes` from module `Unicode` to
```

Comment:
I couldn't get the link to work, suggestions welcome.

Reply:
Did ```[`Unicode.graphemes`](@ref)```not work?

Reply:
No.

Reply:
I think it is fine to leave it like this then until we figure it out,

Reply:
Yeah, anyway I don't have any solution to suggest. Merge then?

Reply:
Sure, perhaps run docs locally and see if it looks OK? Otherwise, looks good to me.

Reply:
They look OK, just the link isn't there (obviously :-).